.fixtures.yml

```diff
@@ -1,9 +1,9 @@
 fixtures:
   repositories:
     stdlib:
-      repo: "git://github.com/puppetlabs/puppetlabs-stdlib"
+      repo: "git@github.com:puppetlabs/puppetlabs-stdlib.git"
```

Comment:
Will this break something if the user (maybe Jenkins, maybe Travis) doesn't have a public key on GitHub?